grammars/rpm-spec.cson

```diff
@@ -505,7 +505,7 @@
   'comments':
     'patterns': [
       {
-        'match': '#.*$',
+        'match': '^\s*#.*$',
```

Comment:
This is a good issue to bring up.  The Microsoft sharp languages invalidate the assumptions made by trivial parsers for languages with an octothorpe for the start-of-line-comment character.  (Microsoft marketing spawned names like .Net also cause parsing problems for other cases, but that is a digression.)

The offered solution is not one that I can accept. This expression means that only lines that start with comments or whitespace plus comments are comments.

In other words, this excludes valid end-of-line comments from appearing anywhere in the file.

I have a partial solution in the work.  However this requires either unclear and clever tricks or working negative negative-width assertions for simplicity.  In any case the required changes will be large enough to force a 0.9.0 version at least.